docs/Configuring-Pantheon/Consensus-Mechanisms.md

```diff
@@ -46,9 +44,19 @@ increases number as the of validators increases.
 
 For IBFT 2.0, the time to add new blocks increases as the number of validators increases.   
 
-### Minimum Number of Validators 
+### Type of Byzantine-fault-tolerance
+
+IBFT 2.0 features a classical BFT consensus protocol that ensures safety (no fork is possible) provided that no more than
+(n-1)/3 of the validators (truncated to the integer value) are Byzantine.
+For example in an IBFT 2.0 network of:
+* 3, no Byzantine nodes are tolerated
+* 4-6, 1 Byzantine node is tolerated
+* 7-9, 2 Byzantine nodes are tolerated
+
+Clique features a probabilistic consensus protocol (à la Nakamoto) where forks are possible but the deeper a block is
```

Comment:
nit: I'd reference ethHash (or Hashimoto) rather than Nakamoto, just to stick with Ethereum concepts.

Reply:
Done

Reply:
Nakamoto in this case refers to the the general category of Consensus Protocols where availability is preferred over consistency. I would leave it as it was

Reply:
Done